doc/api/process.md

```diff
@@ -2601,6 +2601,81 @@ the [`'warning'` event][process_warning] and the
 [`emitWarning()` method][process_emit_warning] for more information about this
 flag's behavior.
 
+## `process.permission`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object}
+
+This API is available through the [`--experimental-permission`][] flag.
+
+`process.permission` is an object whose methods are used to manage permissions
+for the current process. Additional documentation is available in the
+[Permission Model][].
+
+### `process.permission.check(scope[, reference])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean}
+
+* `scopes` {string}
+
+* `reference` {string}
+
+Verifies that the process is able to access the given scope and reference.
+If no reference is provided, a global scope is assumed.
+
+The reference has a meaning based on the provided resource. For example,
+the reference when the scope is File System means files and folders.
+
+The available scopes are:
+
+* `fs` - All File System
+* `fs.read` - File System read operations
+* `fs.write` - File System write operations
+
+```js
+// Check if the process has permission to read the README file
+process.permission.check('fs.read', './README.md');
+// Check if the process has read permission operations
+process.permission.check('fs.read');
+```
+
+### `process.permission.deny(scope[, reference])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean}
+
+* `scope` {string}
+
+* `reference` {string}
+
+Deny permissions at runtime.
+
+The available scope are:
+
+* `fs` - All File System
+* `fs.read` - File System read operations
+* `fs.write` - File System write operations
+
+The reference has a meaning based on the provided resource. For example,
+the reference when the scope is File System means files and folders.
+
+```js
+// Deny READ operations to the ./README.md file
+process.permission.deny('fs.read', './README.md');
+// Deny ALL WRITE operations
+process.permission.deny('fs.write');
```

Comment:
The code requires an array for `references`, but the doc shows a string. Can we fix the discreptency?